test/system/helpers.bash

```diff
@@ -55,6 +55,18 @@ function basic_setup() {
     PODMAN_TMPDIR=$(mktemp -d --tmpdir=${BATS_TMPDIR:-/tmp} podman_bats.XXXXXX)
 }
 
+# Check that an unprivileged user cannot read or write to the storage
+function check_no_unprivileged_access() {
+    GRAPH_ROOT=$(run_podman info --format json | jq .store.GraphRoot)
+    RUN_ROOT=$(run_podman info --format json | jq .store.RunRoot)
+    find $GRAPH_ROOT $RUN_ROOT -perm -o+w | while read i; do
```

Comment:
no I meant that.  I'd prefer to test as many files/directories as possible, if it turns to be too expensive we can tune it

Reply:
What I mean is: can you try running that `find` command manually, with `-ls` instead of `-print`? I think you'll find that there are a lot of symlinks, because all symlinks are mode 777, but that's meaningless. There is no point whatsoever to testing those.

Reply:
a lot of them in my current storage are coming from `busybox`, we could probably try to skip these as we are really testing the same thing multiple times.  I'd still like though to test symlinks and that we don't have access to rm them

Reply:
Let me try another way: can you explain what it is you think you are testing by paying attention to symlinks? I think there's a major misunderstanding going on here. Let's say the `find` produces output like this:
```
/x -> /y
```
What is the purpose of running any tests on `/x`? It is the same as running on `/y`, and:
* if `/y` is a writable file or directory, the `find` command will emit it as part of its output
* if `/y` is not a writable file or directory, there's no point in testing it

Reply:
in the last version I've pushed, I am also checking for `rm`.

In the example you've given `/x -> /y` the unprivileged user could still be able to `rm /x` but not having privileges for doing it on `/y`

Reply:
Permissions on `/y` are irrelevant for `rm`; only the parent directory permissions matter. 

I'm working on a little bit of a rewrite; I'll push it when I'm comfortable with it. It might be in a few hours.

Reply:
and symlink ownership.  But given anyway that we are hardcoding the user, it probably doesn't make sense to test symlinks this way and we can safely skip them